packages/ai-chat/package.json

```diff
@@ -46,6 +46,7 @@
     "**/web-components/**/*.js"
   ],
   "devDependencies": {
+    "@carbon/ai-chat-components": "*",
```

Comment:
so something to keep in mind is that @carbon/ai-chat-components is currently "private". When do we make it public even if in beta?